joining/src/main/scala/akka/cluster/bootstrap/ClusterBootstrapSettings.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.cluster.bootstrap
+
+import java.util.Locale
+
+import akka.actor.ActorSystem
+import com.typesafe.config.Config
+
+import scala.concurrent.duration._
+import scala.concurrent.duration.FiniteDuration
+
+// FIXME make easy to binary evolve (kaze class)
+// FIXME don't hard-code
+// TODO security for the contact point requests?
+// FIXME FIXME !! redesign the settings !! half of those are kubernetes and DNS specific
+/**
+ *
+ * @param serviceName if None the actor system name will be used
+ */
+final case class ClusterBootstrapSettings(
+    bootstrapMethod: String = "dns-svc",
+    serviceName: Option[String] = Some("appka-service"), // k8s specific ??
+    namespaceName: Option[String] = Some("default"), // k8s specific ??
```

Comment:
I think the `default` namespace is Kubernetes specific - Mesos has a different naming convention for its SRV entries - it's documented in `SRV Records` in this link:

https://mesosphere.github.io/mesos-dns/docs/naming.html